src/demo/containers/playground/FormServicesIntegrationDemo.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react'
+import { TokenContext } from '../App'
+import { FormGroup, FormData, ListRequest, ListResponse, StatusEnum } from 'lib/utils/jsonResponses/Forms'
+import { SynapseClient } from 'lib'
+import { FileUploadComplete } from 'lib/utils/jsonResponses/FileUploadComplete'
+
+type FormServicesIntegrationDemoState = {
+  token?: string
```

Comment:
[here's a link](https://reactjs.org/docs/context.html#classcontexttype) to the react docs showing how to use the context value from the class itself, e.g. `this.context`, so you don't have to copy the context value over to state

Reply:
I'll check it out!

Reply:
Tried setting
FormServicesIntegrationDemo.contextType = TokenContext
but this.context was not updated with the token (was always just {} after login or session init)

Reply:
Okay, we can leave this as a TODO for now and come back to it later